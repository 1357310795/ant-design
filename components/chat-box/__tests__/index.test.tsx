import ChatBox from '..';
import mountTest from '../../../tests/shared/mountTest';
import rtlTest from '../../../tests/shared/rtlTest';

describe('chat-box', () => {
  mountTest(ChatBox);
  rtlTest(ChatBox);
});
