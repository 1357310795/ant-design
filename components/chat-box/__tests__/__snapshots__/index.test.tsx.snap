// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chat-box rtl render component should be rendered correctly in RTL direction 1`] = `
<div
  class="ant-chatbox ant-chatbox-start ant-chatbox-rtl"
>
  <div
    class="ant-chatbox-content"
  />
</div>
`;
